packages/extension-base/src/services/inapp-notification-service/index.ts

```diff
@@ -252,7 +288,75 @@ export class InappNotificationService implements CronServiceInterface {
         title: NotificationTitleMap[actionType].replace('{{tokenSymbol}}', symbol),
         description: NotificationDescriptionMap[actionType](formatNumber(amount, decimals), symbol),
         time: timestamp,
-        extrinsicType: ExtrinsicType.CLAIM_AVAIL_BRIDGE,
+        extrinsicType: ExtrinsicType.CLAIM_BRIDGE,
+        isRead: false,
+        actionType,
+        metadata
+      };
+    });
+
+    await this.validateAndWriteNotificationsToDB(notifications, address);
+  }
+
+  // Polygon Claimable Handle
+
+  // @ts-ignore
+  async createPolygonClaimableTransactions () {
+    const { evm: evmAddresses } = this.getCategorizedAddresses();
+    const etherChains = [COMMON_ASSETS.ETH, COMMON_ASSETS.ETH_SEPOLIA];
```

Comment:
Not handle for case L1 -> L2. With the result of `fetchPolygonBridgeTransactions`, have any case return data of L1 -> L2?

Reply:
No data with status `READY_TO_CLAIM` on case bridge from L1 -> L2